src/main/java/com/github/redouane59/twitter/TwitterClient.java

```diff
@@ -131,6 +132,11 @@ public TwitterClient(TwitterCredentials credentials, OAuth10aService service) {
     requestHelperV2    = new RequestHelperV2(credentials, service);
   }
 
+  public void setAPIListener(IAPIEventListener listener) {
```

Comment:
Can't you set `IAPIEventListener listener` as an argument of the `startFilteredStream()` method instead ? Just asking but maybe there is something I didn't see.

Reply:
Sure it could be made with startFilteredStream with 2 different signatures : 
startFilteredStream(Consumer<Tweet> consumer) 
startFilteredStream(IAPIEventListener listener).

it would answer the question above. :)